README.md

```diff
@@ -1,5 +1,8 @@
 # Kotlin Client for PostgREST
 
+> **Warning**
+> This repository is archived. Use [supabase-kt](https://github.com/supabase-community/supabase-kt) instead to use Supabase in your Android application.

```

Comment:
Kotlin isn't exclusive for android, it can be used as a direct replacement for Java on the backend or even for native apps. The new kotlin lib is multiplatform, will rephrase tomorrow and merge 🙏

Reply:
I could rephrase it right now. Would something like this be more appropreate?

```suggestion
> This repository is archived. Use [supabase-kt](https://github.com/supabase-community/supabase-kt) instead to use Supabase in your Kotlin project.

```